bindgen-cli/options.rs

```diff
@@ -362,14 +363,17 @@ struct BindgenCommand {
     /// Generate wrappers for `static` and `static inline` functions.
     #[arg(long, requires = "experimental")]
     wrap_static_fns: bool,
-    /// Sets the path for the source file that must be created due to the presence of `static` and
-    /// `static inline` functions.
+    /// Sets the path of the file where generated native code will be emitted.
     #[arg(long, requires = "experimental", value_name = "PATH")]
-    wrap_static_fns_path: Option<PathBuf>,
+    native_code_generation_path: Option<PathBuf>,
     /// Sets the suffix added to the extern wrapper functions generated for `static` and `static
     /// inline` functions.
     #[arg(long, requires = "experimental", value_name = "SUFFIX")]
     wrap_static_fns_suffix: Option<String>,
+    /// Create a wrapper function for the macro. The MACRO value must be of the shape
+    /// `[<return type>] <macro name>[(<comma separated list of arguments>)]`.
+    #[arg(long, value_name = "MACRO")]
+    macro_function: Option<Vec<String>>,
```

Comment:
I suspect this should require the `experimental` feature as it leverages the infrastructure for `--wrap-static-fns` which is itself experimental